dependencies.gradle

```
@@ -4,7 +4,7 @@ ext {
     targetSDK = 29
     compileSDK = 29
 
-    buildTools = '4.1.0-alpha01'
+    buildTools = '4.1.0-alpha08'
```

Atualizar para versão abaixo para rodar os testes corretamente
```suggestion
    buildTools = '4.1.1'
```